channeldb/channel_test.go

```diff
@@ -612,8 +614,10 @@ func TestChannelStateTransition(t *testing.T) {
 			LogIndex:      uint64(i * 2),
 			HtlcIndex:     uint64(i),
 		}
-		htlc.OnionBlob = make([]byte, 10)
-		copy(htlc.OnionBlob[:], bytes.Repeat([]byte{2}, 10))
```

Comment:
Q: Wondering where this testcase came from, I think onion-blobs were always 1366 in size also before this change ?